gitarro.rb

```diff
@@ -8,7 +8,9 @@
 require_relative 'lib/gitarro/backend'
 
 b = Backend.new
-prs = b.open_prs
+prs = b.open_newer_prs
+exit 0 if b.check_pr_list_empty(prs)
```

Comment:
following issue #64 we can rename this as
`prs_list_empty?`

Reply:
Done, pending test and push.